common/scala/src/main/scala/whisk/core/entity/Limits.scala

```diff
@@ -67,7 +67,15 @@ protected[core] object ActionLimits
     /** Creates a ActionLimits instance with default duration and memory limit. */
     protected[core] def apply(): ActionLimits = ActionLimits(TimeLimit(), MemoryLimit())
 
-    override protected[core] implicit val serdes = jsonFormat3(ActionLimits.apply)
+    override protected[core] implicit val serdes = new RootJsonFormat[ActionLimits] {
+        val helper = jsonFormat3(ActionLimits.apply)
+
+        def read(value: JsValue) = {
+            val inter = helper.read(value)
+            inter.copy(logs = Some(inter.logs.getOrElse(LogLimit())))
```

Comment:
the dual  - since you have a custom serdes - is to make ActionLimit use a definite LogLimit (not an Option) and tolerate the missing LogLimit in the serializer.